packages/datagateway-common/src/table/columnFilters/dateColumnFilter.component.tsx

```diff
@@ -73,16 +73,14 @@ const DateColumnFilter = (props: {
       <MuiPickersUtilsProvider utils={DateFnsUtils}>
         <KeyboardDatePicker
           style={{ whiteSpace: 'nowrap' }}
-          clearable
-          inputProps={{ 'aria-label': `${props.label} filter from` }}
+          inputProps={{ 'aria-label': `${props.label} filter` }}
```

Comment:
@sam-glendenning @AkhilMag13 I am adding the "from" that was removed in this line back in, otherwise the same aria-label is shared for both the "from" and "to" inputs.

Reply:
There is a problem with this - the from and to are repeated twice when the screen reader is on the datepickers. It reads it out as: "Start Date filter from edit  From...  blank"